src/commands/database/database.ts

```diff
@@ -0,0 +1,43 @@
+import BaseCommand from '../base-command.js'
+import { status } from './status.js'
+import { init } from './init.js'
+
+export type Extension = {
+  id: string
+  name: string
+  slug: string
+  hostSiteUrl: string
+  installedOnTeam: boolean
+}
+
+export type SiteInfo = {
+  id: string
+  name: string
+  account_id: string
+  admin_url: string
+  url: string
+  ssl_url: string
+}
+
+export const createDatabaseCommand = (program: BaseCommand) => {
+  const dbCommand = program
+    .command('db')
+    .alias('database')
+    .description(`Provision a production ready Postgres database with a single command`)
+    .addExamples(['netlify db status', 'netlify db init', 'netlify db init --help'])
+
+  dbCommand
+    .command('init')
+    .description(`Initialize a new database for the current site`)
+    .option(`--drizzle`, 'Initialize basic drizzle config and schema boilerplate')
```

Comment:
I would instead make this something like `--boilerplate=drizzle` and make `drizzle` the default. This will make it so we can easily change the list of tools we support later on without adding tons of flags. Eventually, one of the new flags we want to add is going to conflict with another flag.